controlsfx/src/main/java/org/controlsfx/control/GridView.java

```diff
@@ -146,7 +157,39 @@ public GridView(ObservableList<T> items) {
     @Override public String getUserAgentStylesheet() {
         return getUserAgentStylesheet(GridView.class, "gridview.css");
     }
-    
+
+    /**
+     * Programmatically scrolls the GridView to display the cell for the given index.
+     * If the GridView has not been rendered yet (e.g., it's not on a visible stage),
+     * this method may have no effect.
+     *
+     * @param index The index of the item to scroll to.
+     */
+    public void scrollTo(int index) {
+        if (getItems() == null || getItems().isEmpty() || getColumnCount() == 0) {
+            return;
+        }
+        int newIndex = Utils.clamp(0, index, getItems().size() - 1);
+        int rowIndex = newIndex / getColumnCount();
+        Event.fireEvent(this, new ScrollToEvent<>(this, this, ScrollToEvent.scrollToTopIndex(), rowIndex));
+    }
+
+    /**
+     * Programmatically scrolls the GridView to display the cell for the given item.
+     * If the item is not found or the GridView has not been rendered yet, this method
+     * may have no effect.
+     *
+     * @param item The item to scroll to.
+     */
+    public void scrollTo(T item) {
+        if( getItems() != null ) {
```

Comment:
code formatting looks weird here

Reply:
Thank you for pointing that out! 🙏
I’ve fixed the formatting issues